libs/cart/src/drivers/magento/models/outputs/cart.ts

```diff
@@ -22,6 +22,14 @@ export interface MagentoCart {
     grand_total: MagentoMoney,
     subtotal_excluding_tax: MagentoMoney,
     subtotal_including_tax: MagentoMoney,
-    subtotal_with_discount_excluding_tax: MagentoMoney,
+		subtotal_with_discount_excluding_tax: MagentoMoney,
+		applied_taxes: {
+			amount: MagentoMoney,
+			label: string
+		}[],
+		discounts: {
```

Comment:
Are you sure that `applied_taxes` and `discounts` are non-nullable? Magento has an annoying habit of returning `null` for values that should be empty arrays.

Reply:
Testing in postman looks like `discounts` will be `null` if none exist.

Reply:
you're probably right. fixed